backend/spec/requests/activities_controller_spec.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'ActivitiesController', type: :request do
+  let(:user) { create(:user) }
+
+  describe 'GET' do
+    context 'when index' do
```

Comment:
We usually would use  `describe 'GET index'` and `context` more in for all different contexts/szenarios such as "when not logged in". But its more of an attitude, and i am fine with this :+1:

Reply:
*ignore